src/app/util/util.h

```diff
@@ -74,8 +74,7 @@
 // Cluster name structure
 typedef struct
 {
-    uint16_t id;
-    uint16_t mfgCode;
+    uint32_t id;
```

Comment:
Shouldn't this be `ClusterId`?

Reply:
My point is that assuming that "cluster id" is "uint32_t" is bad and we should not do it.

Reply:
Ah, data-type wise? Sure, that I believe I can change (assuming includes work)